packages/workflow/src/Interfaces.ts

```diff
@@ -124,6 +124,8 @@ export interface ICredentialsDecrypted {
 	type: string;
 	nodesAccess: ICredentialNodeAccess[];
 	data?: ICredentialDataDecryptedObject;
+	ownedBy?: IDataObject;
```

Comment:
Should we perhaps move the `UserSharingsDetails` type to the `workflow` package and link it here instead of using `IDataObject`? This can then be easily shared with FE for uniformity.